projects/components/src/navigation/navigation-list.component.scss

```diff
@@ -84,5 +84,5 @@
 }
 
 .expanded {
-  width: 236px;
```

Comment:
Can you please check if all areas look fine after this change?
- Main Nav
- Settings Nav
- Self Service Nav

Reply:
yep. all 3 looks fine.

Reply:
okay